examples/golang/docker-compose-go.yaml

```diff
@@ -0,0 +1,18 @@
+version: "3.8"
+
+services:
+  go_plugin_compile:
+    build:
+      context: ../shared/build
+    command: >
+      bash -c "
+      bazel build  //examples/golang/simple:simple.so
+      && cp -a ./bazel-bin/examples/golang/simple/simple.so /output"
+    environment:
+    - BUILD_UID=${UID:-1000}
```

Comment:
not sure why - this doesnt seem to be working - the built `lib` folder doesnt have my host user's permissions

Reply:
May it be related to the Docker or docker-compose version?

Reply:
its missing the `entrypoint.sh` with gosu

you can either mv the entrypoint from `wasm-cc` -> `shared/build` or copy it here

Reply:
Done.  https://github.com/envoyproxy/envoy/pull/25284